WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1343,6 +1343,18 @@ decisions = {
 				factor = 0
 				religion = marcionism
 			}
+			modifier = {
+				factor = 10
+				trait = kind
+			}
```

Comment:
No. I mean that there are differences between yourself and myself in terms of Judaism in the Medieval period.